src/test/java/ua/edu/internship/interview/service/business/UserQuestionServiceTest.java

```diff
@@ -0,0 +1,166 @@
+package ua.edu.internship.interview.service.business;
+
+import org.bson.types.ObjectId;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import ua.edu.internship.interview.data.documents.SkillDocument;
+import ua.edu.internship.interview.data.documents.UserQuestionDocument;
+import ua.edu.internship.interview.data.enumeration.QuestionDifficulty;
+import ua.edu.internship.interview.data.enumeration.QuestionType;
+import ua.edu.internship.interview.data.repository.SkillRepository;
+import ua.edu.internship.interview.data.repository.UserQuestionRepository;
+import ua.edu.internship.interview.service.dto.skill.SkillDto;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionCreateDto;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionDto;
+import ua.edu.internship.interview.service.dto.user.question.UserQuestionUpdateDto;
+import ua.edu.internship.interview.service.mapper.UserQuestionMapper;
+import ua.edu.internship.interview.service.utils.exceptions.NoSuchEntityException;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:ChAxMWNkYTdmM2E5YTQxYjY1EgpjaGVja3N0eWxl -->